### PR TITLE
Lock all

### PR DESCRIPTION
**Pull request by kepta**
This adds a lock/unlock/status update api for multiple items

cc @batpad

### CODE REVIEW

File: routes/item.js [hunk 1]
```diff
@@ -430,41 +413,113 @@ function updateItem(req, res, next) {
...
- * @param {String} opts.project - the id of the project the item belongs to
- * @param {String} opts.item - the id of the item itself
- * @param {String} opts.user - the user making the change
- * @param {FeatureCollection} [opts.featureCollection] - a validated GeoJSON feature collection, required on create
- * @param {Point} [opts.pin] - a validated GeoJSON point representing the queryable location of this item, required on create
- * @param {String} [opts.instructions] - the instructions for what needs to be done, required on create
- * @param {String} [opts.status] - the status to set the item to
- * @param {String} [opts.lockedTill] - the time the lock expires at
+ * Updates an array of items. Note: max limit is 500
+ * @name update-all-item
```
Comment (batpad) [inline routes/item.js]: can this be `update-all-items` ?

Comment (kepta) [reply routes/item.js]: oops, fixing it.

**Comment by batpad** [comment]
This looks great to me @kepta !

**Comment by batpad** [comment]
@kepta one thing: can we add logging to this end-point?

See: https://github.com/osmlab/to-fix-backend/blob/master/routes/item.js#L392 and https://github.com/osmlab/to-fix-backend/blob/master/routes/item.js#L424

**Pull request merged**